base/doc/ltnews37.tex

```diff
@@ -149,6 +149,29 @@ \section{New or improved commands}
 %
 \githubissue{xxx}
 
+\subsection{Extension of the \cs{label} command}
+
+Up to now the \cs{label} command stored in standard \LaTeX{} into the
+\cs{newlabel} command in the \texttt{.aux} file two data: 
```

Comment:
would use "values" not "data" even if data is correct in its plural

Reply:
or "data items"